core/src/main/java/org/opensearch/sql/data/model/ExprTimeValue.java

```diff
@@ -53,7 +64,7 @@ public ExprTimeValue(String time) {
 
   @Override
   public String value() {
-    return DateTimeFormatter.ISO_LOCAL_TIME.format(time);
+    return time.format(FORMATTER_ZERO_PAD_NANOS);
```

Comment:
This change was an experiment. Makes second fraction always printed with precision 6 digits with zero padding.
It could be helpful for cases when user wants to parse DateTIme.